app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -19,6 +19,14 @@ def insecure
     registration_params['insecure'] ? '--insecure' : ''
   end
 
+  def is_token_valid(min_value, max_value)
```

Comment:
```suggestion
  def expiration_valid?(min_value, max_value)
```

This method doesn't check the token itself for validity, only the expiration value, so let's call it like that.
Additionally, it's custom in Ruby to use `something?` (ending in a question mark) to denote methods that check things.